content/national-qualifications-framework/ncit/content/write-a-report/_index.md

```diff
@@ -47,31 +47,31 @@ title: Write a report
 
 ### Key features
 
 Common features of technical reports include:
 
 
- - Designed for quick and easy communication of information
- - Designed for selective reading (a manager or client may only read a section of the report depending on what they are interested in learning)
- - Have sections with numbered headings and subheadings
- - Use figures and diagrams to convey data
- - Based primarily on original research
+ - Designed for quick and easy communication of information.
+ - Designed for selective reading (a manager or client may only read a section of the report depending on what they are interested in learning).
+ - Have sections with numbered headings and subheadings.
+ - Use figures and diagrams to convey data.
+ - Based primarily on original research.
   - If you use information, ideas, media, etc that someone else has created, you must properly cite your sources with footnotes.
   - Failure to cite the sources of ideas in your report is considered plagiarism. This violates professional ethics and will result in you losing respect among peers and can even lead to the loss of a job.
 
 ### Collecting and organising your information
 
-Based on the research you did during your HCD interviews and observations, you should have a lot information and insights that you can use to put into a report. Review the notes, photos, and other documents you made during the Empathise/Research phase and group them into big ideas or topics (review this section of the HCD curriculum if you are unclear what to do). Once you’ve figured out all the topics you want to cover, start to think about a logical order to present these ideas. One good way to organise your ideas is to create an outline.
+Based on the research you did during your HCD (Human Centered Design) interviews and observations, you should have a lot information and insights that you can put into a report. Review the notes, photos, and other documents you made during the Empathise/Research phase and group them into big ideas or topics (review this section of the HCD curriculum if you are unclear what to do). Once you’ve figured out all the topics you want to cover, start to think about a logical order to present these ideas. One good way to organise your ideas is to create an outline.
 
 ## 2. Outline and structure
 
-The first step in writing any report is to create an outline. An outline breaks down the parts of your report in a clear, hierarchical manner. You should always create an outline before you start writing, because it will help you organise your thoughts. If your outline is good, your paper should be easy to write.
+The first step in writing any report is to create an outline. An outline breaks down the parts of your report in a clear and hierarchical manner. You should always create an outline before you start writing because it will help you organise your thoughts. If your outline is good then your paper should be easy to write.
```

Comment:
```suggestion
The first step in writing any report is to create an outline. An outline breaks down the parts of your report in a clear and hierarchical manner. You should always create an outline before you start writing because it will help you organise your thoughts. If your outline is good, then your paper should be easy to write.
```